osu.Game.Tests/Gameplay/TestSceneStoryboardSamples.cs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using osu.Framework.Audio;
+using osu.Framework.Audio.Sample;
+using osu.Framework.IO.Stores;
+using osu.Game.Audio;
+using osu.Game.Skinning;
+using osu.Game.Tests.Resources;
+using osu.Game.Tests.Visual;
+
+namespace osu.Game.Tests.Gameplay
```

Comment:
Was this meant to land in this namespace instead of `osu.Game.Tests.Visual.Gameplay`? This one makes `Visual` prefixes to show up everywhere else.

Reply:
this should've been `HeadlessTest` since it doesn't seem to have visuals anywhere.